source/howtos/dynamic_services.rst

```diff
@@ -0,0 +1,99 @@
+Dynamic Services
```

Comment:
This is a v good idea for a howto -- was wondering about the details of this myself